### DISCUSSION TITLE
ELKS v0.6.0 is working! 😃

### DISCUSSION THREAD

**Discussion by DartPower** [copy/v86#682] [General]
[ELKS v0.6.0](https://github.com/jbruchon/elks/releases/tag/v0.6.0)
I'm tested it and it's works! Can you add this to list? 😃 
But im tested only "[hd32mbr-fat.img](https://github.com/jbruchon/elks/releases/download/v0.6.0/hd32mbr-fat.img)" and i think other images can too work.

**Comment by copy** [comment]
I recently made an experimental service to upload user images, could you try it? https://v86-upload.copy.sh/

**Comment by DartPower** [comment]
Hard drive image: https://github.com/jbruchon/elks/releases/download/v0.6.0/hd32mbr-fat.img
Bad image file (signature)

But this OK: https://github.com/jbruchon/elks/releases/download/v0.6.0/fd2880-fat.img
```
Ok ... DartPower ELKSv060 fda 128 8 https://copy.sh/v86/?profile=DartPower/ELKSv060
{"fda":{"url":"file.bin","size":2949120,"async":false},"memory_size":134217728,"vga_memory_size":8388608,"acpi":false,"id":"ELKSv060","name":"DartPower/ELKSv060"}
```
